calicoctl/calicoctl.go

```diff
@@ -35,6 +35,7 @@ func main() {
     get            Get a resource identified by file, stdin or resource type and name.
     version        Display the version of calicoctl.
     node           Node related commands.
+    checksystem    Check for incompatibilities on the host system.
```

Comment:
yup! I had it worded like that before, but then I realized python version has it that way so I changed it. I'll change it back to more positive